### PR TITLE
Update accessing-boards.rst

### PR DESCRIPTION
**Pull request by justinegeffen**
Added icon for Product Menu based on user feedback.

### CODE REVIEW

**Comment by cwarnermm** [comment]
Thanks, @justinegeffen! Could you add an inline Boards icon image as well for step #2, please, given that there's 2 ways to get to boards and users would benefit from seeing visual examples of both.

File: source/boards/accessing-boards.rst [hunk 1]
```diff
@@ -18,7 +18,12 @@ Access your boards
   :target: https://mattermost.com/deploy
   :alt: Available for Mattermost Self-Hosted deployments.
 
-You can access all your boards as well as boards in channels you belong to in the Boards dashboard. Select **Boards** from the Product menu in the top-left corner of Mattermost to open the dashboard and switcher. You can also select the Boards icon in the channel header to open the board for the current channel, direct message, or group message.
+Select **Boards** from the **Product menu** |product-list| in the top-left corner of Mattermost to access all your boards as well as boards in channels you belong to.
```
Comment (justinegeffen) [inline source/boards/accessing-boards.rst:21]:
```suggestion
Select **Boards** from the **product menu** |product-list| in the top-left corner of Mattermost to access all your boards as well as boards in channels you belong to.
```

File: source/playbooks/get-started-with-playbooks.rst [hunk 2]
```diff
@@ -1,7 +1,7 @@
 Get started with Mattermost Playbooks
 =====================================
 
-Access Mattermost Playbooks by selecting **Playbooks** from the **Product list** |product-list| icon.
+Access Mattermost Playbooks by selecting **Playbooks** from the **Product menu** |product-list|.
```
Comment (justinegeffen) [inline source/playbooks/get-started-with-playbooks.rst:4]:
```suggestion
Access Mattermost Playbooks by selecting **Playbooks** from the **product menu** |product-list|.
```

**Review by cwarnermm — APPROVED** [review]
Thanks!

**Pull request merged**